Updates to combustion symposium workshop info

rwest opened the pull request:
Mostly small changes.

rwest committed:
Change the call to action to register. Fix grammar typo.

rwest committed:
Details on venue

rwest committed:
Update materials link. Also put into future tense.

rwest committed:
Soliciting suggestions.

We should say HOW to contact the organizers.
A workshops@cantera.org email address would be nice.

rwest committed:
Tweak the "attending" text on the participants page.

rwest committed:
Add China to the list of hosts (though no bio exists yet)

rwest committed:
Tweaking the homepage

kyleniemeyer commented:
@rwest looks good! Also add Patrick to the list of hosts? 

@chinahg could you share a bio here we can use?

rwest committed:
Added Patrick Meagher

rwest committed:
Added bio for @chinahg

I saw kramdown mentioned in _config.yml so am hoping
that the kramdown extension to markdown will work to
specify the anchor id
https://kramdown.gettalong.org/syntax.html
else the link might not work right

rwest commented:
Added Patrick and China.  Please could someone check and merge?

rwest committed:
Tweaking the registration call-to-action link
